content/en/logs/guide/detect-unparsed-logs.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Monitor and query for unparsed logs
+kind: guide
+aliases:
+  - /logs/guide/detect-unparsed-logs/
```

Comment:
```suggestion
```
You need aliases only if there's a 404 you're trying to avoid by redirecting a deleted page to this page. I think we can delete these lines in this file